src/questions/molecule.yaml

```diff
@@ -0,0 +1,136 @@
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: They cannot be divided
+      key: A
+    - content: They can be divided into smaller fractions
+      key: B
+    - content: They can only be divided into two parts
+      key: C
+    - content: They can be divided into fractions but cannot be distributed for ownership.
+      key: D
+- uuid: 0f517eb7-fa3c-4386-a239-6150b94bb9eb
+  type: SingleChoice
+  content: What is the significant risk associated with traditional IP?
+  hint: NoHint
+  explanation: One significant risk associated with traditional IP is the possibility of a dispute over control, both due to inadequate default IP laws and jurisdictional conflicts.
+  answerKeys:
+    - B
+  difficultyLevel: Low
+  choices:
+    - content: Web3 experimentation
+      key: A
+    - content: Dispute over control
+      key: B
+    - content: Liability from economic experimentation
+      key: C
+    - content: Poor choices about when or whether to file a patent
+      key: D
+- uuid: 3fd8e9f4-0d77-4132-8a2e-01961ad563c8
+  type: SingleChoice
+  content: What are some of the potential risks associated with using IP-NFTs to commercialize intellectual property assets?
+  hint: NoHint
+  explanation: Exposure of IP to new risks and liability from economic experimentation are some of the potential risks associated with using IP-NFTs to commercialize intellectual property assets.
+  answerKeys:
+    - D
+  difficultyLevel: Low
+  choices:
+    - content: Inadequate default IP laws and jurisdictional conflicts leading to disputes over control
+      key: A
+    - content: Difficulty in determining when and whether to file a patent on IP-NFTs 
+      key: B
+    - content: Limited ability to fractionalize ownership and governance of IP-NFTs
+      key: C
+    - content: Exposure of IP to new risks and liability from economic experimentation
+      key: D
+- uuid: ca6c4de1-1b60-404d-8f46-321176e66c45
+  type: SingleChoice
+  content: What is the primary goal of Molecule Discovery?
+  hint: NoHint
+  explanation: Molecule Discovery is a platform that features various biopharma research projects from across the globe. Researchers present their projects in different therapeutic areas. The platform allows you to sort projects based on clinical stages and opportunity types.
+  answerKeys:
+    - C
+  difficultyLevel: Low
+  choices:
+    - content: To create web3-enabled wallets for biopharma researchers
+      key: A
+    - content: To tokenize intellectual property rights using non-fungible tokens
+      key: B
+    - content: To assist biopharma researchers in finding resources and partners for their research projects in a decentralized manner
+      key: C
+    - content: To educate the community on different therapeutic areas
+      key: D
+- uuid: 6df29c6a-0156-46b2-96fe-93c0835794ac
+  type: SingleChoice
+  content: Which of the following is NOT a way to create an account on Molecule Discovery?
```

Comment:
We can drop this question as this doesn't add too much value to the reader's knowledge